docs/proposals/policy.md

```diff
@@ -0,0 +1,146 @@
+
```

Comment:
Please structure this as brief description of problem, goals, non-goals, approaches (standard proposal form)